README.md

```diff
@@ -75,5 +77,5 @@ As of version 1.0.0 we completely removed the `Entypo` font set!
 This project was inspired by [this dribbble shot](http://dribbble.com/shots/1072278) and [this cssdeck experiment](http://cssdeck.com/labs/css-social-share-button) - huge props to these two guys for some incredible ideas and work.
 
 # Contributing and License
-- Contributing Guidelines can be found [here](contributing.md)
-- Licenced under MIT - [details here](license.md)
+- Contributing Guidelines can be found [here](CONTRIBUTING.md)
+- Licenced under MIT - [details here](LICENSE.md)
```

Comment:
also `s/Licenced/Licensed/`... just because you're using 2 different spellings here